scripts/load_cql.py

```diff
@@ -154,42 +153,61 @@ def execute_commands(cursor, commands, verbose):
                 print("Ok.")
 
 
-def setup_connection(reactor, args):
-    """
-    Return Cassandra connection
-    """
-    return CQLClient(
-        clientFromString(reactor, 'tcp:{}:{}'.format(args.host, args.port)),
-        args.keyspace)
-
-
-def webhook_index(reactor, args):
+def webhook_index(reactor, conn):
     """
     Show webhook indexes that is not there table connection
     """
     store = CassScalingGroupCollection(None, None)
     eff = store.get_webhook_index_only()
-    conn = setup_connection(reactor, args)
-    return perform(get_cql_dispatcher(reactor, conn), eff).addCallback(
-        pprint).addCallback(lambda _: conn.disconnect())
+    return perform(get_cql_dispatcher(reactor, conn), eff)
 
 
-def webhook_migrate(reactor, args):
+def webhook_migrate(reactor, conn):
     """
     Migrate webhook indexes to table
     """
     store = CassScalingGroupCollection(None, None)
     eff = store.get_webhook_index_only().on(store.add_webhook_keys)
+    return perform(get_cql_dispatcher(reactor, conn), eff)
+
+
+@inlineCallbacks
+def insert_deleting_false(reactor, conn):
+    """
+    Insert false to all group's deleting column
+    """
+    groups = yield get_scaling_groups(conn)
+    query = (
+        'INSERT INTO scaling_group ("tenantId", "groupId", deleting) '
```

Comment:
Yes. But still need to insert in each row separately. There is no way to insert to all rows. `INSERT` needs the primary key part of the table. This is why the code gets all the groups.